### PR TITLE
Microsoft teams chatbot

### PR DESCRIPTION
**Pull request by DonMartin76**

### CODE REVIEW

File: src/config-updater.js [hunk 1]
```diff
@@ -207,8 +208,37 @@ function saveKickstarter(config, kickData) {
...
 
+function updateStep21_v1_0_0i(targetConfig, sourceConfig, configKey) {
+    debug("Performing updateStep21");
+
+    const targetGlobals = loadGlobals(targetConfig);
+    targetGlobals.version = 21;
+
+    const events = targetGlobals.chatbot.events;  // Copy old events to each new hook
+    // Create new field with target values
+    targetGlobals.chatbot.targets = [];
```
Comment (DonMartin76) [inline src/config-updater.js]: Even if the target version is below `21`, please always check if the new property is already in the configuration. All other steps also work like this - if they have run before, they will not change anything. As it is right now, re-running the update step (e.g. by resetting the `version` property) will completely delete the chatbot configuration.

**Review by DonMartin76 — COMMENTED** [review]
See inline comments.

**Comment by DonMartin76** [comment]
@Jabb0 Please also sign the CLA.

File: src/config-updater.js (hunk 1 above)
Comment (Jabb0) [reply src/config-updater.js]: Code now checks if "targets" is present and if that is the case just update the version number.

**Pull request merged**